openshift-serverless-clients.spec

```diff
@@ -4,7 +4,7 @@
 %global product_name OpenShift Serverless
 %global golang_version 1.14
```

Comment:
If you need a specific version of go, you can set `golang_version` to `1.14.12`.